code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -495,3 +497,79 @@
 
 /obj/item/crusher_trophy/wolf_ear/on_mark_detonation(mob/living/target, mob/living/user)
 	user.apply_status_effect(/datum/status_effect/speed_boost, 1 SECONDS)
+
+//cosmetic items for changing the crusher's look
+
+/obj/item/crusher_trophy/retool_kit
+	name = "crusher sword retool kit"
+	desc = "A toolkit for changing the crusher's appearance without affecting the device's function. This one will make it look like a sword."
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "retool_kit"
+	denied_type = /obj/item/crusher_trophy/retool_kit
+	var/retool_icon = "crusher_sword"
+	var/retool_inhand_icon = "crusher_sword"
+	var/retool_projectile_icon = "pulse1"
+
+/obj/item/crusher_trophy/retool_kit/effect_desc()
+	return "the crusher to have the appearance of a sword"
+
+/obj/item/crusher_trophy/retool_kit/add_to(obj/item/kinetic_crusher/H, mob/user)
```

Comment:
can u change the single letter var here and for the proc below it?

Reply:
the single letter var is also used for the legion skull and demon claws trophies (and we mostly just used it for consistency). should I change those too?

Reply:
would be nice but i wont force u to change the others